packages/lesswrong/components/common/AnalyticsClient.tsx

```diff
@@ -25,6 +27,31 @@ export const AnalyticsClient = () => {
     // There may be events waiting for the client context vars to be set, so flush them now
     flushClientEvents(true);
   }, [currentUserId, clientId, currentUser, abTestGroupsUsed]);
+
+  // Fire a one-time per-tab lifecycle event when a new tab/app instance starts
+  useEffect(() => {
+    // Wait until currentUser has finished loading before firing
+    if (currentUserLoading) return;
+    
+    const tabId = window.tabId;
+    if (!tabId) return;
+    const firedKey = `tabStartedFired:${tabId}`;
+    if (sessionStorage.getItem(firedKey)) return;
+
+    const userAgent = navigator.userAgent ?? null;
+    const abTestGroups = getAllUserABTestGroups(currentUser ? { user: currentUser } : { clientId });
+
+    captureEvent("tabStarted", {
+      tabId,
+      userAgent,
+      abTestGroups,
+    });
+    sessionStorage.setItem(firedKey, "1");
```

Comment:
This should be moved up to right after the sessionStorage check, to minimize race conditions